modules/core/include/opencv2/core/hal/intrin_neon.hpp

```diff
@@ -2271,9 +2269,13 @@ inline v_int32x4 v_round(const v_float32x4& a)
 #endif
 inline v_int32x4 v_floor(const v_float32x4& a)
 {
+#if CV_NEON_AARCH64
+    return v_int32x4(vcvtmq_s32_f32(a.val));
+#else
```

Comment:
The instruction is available on aarch32 too https://developer.arm.com/architectures/instruction-sets/intrinsics/vcvtmq_s32_f32

Reply:
I was struggled at armv7 support. There is no existing macros indicating if target supports only armv7.

Reply:
There is manual "CAROTENE_NEON_ARCH" option. Looks like we need build check or handle march option correctly.

Reply:
> need build check or handle march option correctly.

That sounds like a dedicated PR. How about guard it (as well as the above one) with `CV_NEON_AARCH64` for now?